packages/navigation-menu/src/lib/navigation-menu/components/index.ts

```diff
@@ -0,0 +1,15 @@
+import { MenuComponent } from './menu/menu.component';
+import { MenuTabComponent } from './menu-tab/menu-tab.component';
+import { MenuLinkComponent } from './menu-link/menu-link.component';
+import { NavigationPaneComponent } from './navigation-pane/navigation-pane.component';
+import { SubMenuComponent } from './sub-menu/sub-menu.component';
+import { SubMenuItemComponent } from './sub-menu-item/sub-menu-item.component';
+
+export const COMPONENTS = [
```

Comment:
You must be mistaken since I took this way of working from several examples in the ACPAAS UI Lib. ( Agenda, Avatar, Calendar, Code Snippet, Forms, ...) If you're talking about the casing, well globals consts could be in capitals. Also, looking closer to the acpaas, you'll notice that the whole lib isn't actually consistent at all when it comes to exports....

Reply:
I am mistaken. My bad!